fulp_modules/Z_edits/race_bans/bans.dm

```diff
@@ -1,10 +1,19 @@
-
+/// Case 1: Mob Login. Species are set before mobs are given a mind so we cant just use on_species_gain()
 /mob/sync_mind()
 	. = ..()
-	var/mob/living/carbon/human/H = src
-	H?.dna?.species.check_banned(H)
+	if(ishuman(src))
+		var/mob/living/carbon/human/H = src
+		H?.dna?.species.check_banned(H)
```

Comment:
I’m swagging out

Reply:
Making code easier to understand because DM coders are dumb stupid idiots 💪